charts/backstage/Chart.yaml

```
@@ -44,4 +44,4 @@ sources:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.29.0
+version: 2.29.1
```

```suggestion
version: 3.0.0
```

This would be a breaking change compared to the previous chart version. So I would suggest updating the version accordingly. It should not have any effect on the downstream chart, which would still be 1.5.0